modules/effects/arrow-effects-rx2-extensions/src/main/kotlin/arrow/effects/rx2/extensions/observablek.kt

```diff
@@ -149,6 +163,83 @@ interface ObservableKEffect : Effect<ForObservableK>, ObservableKAsync {
     fix().runAsync(cb)
 }
 
+interface ObservableKConcurrent : Concurrent<ForObservableK>, ObservableKAsync {
+  override fun <A> async(fa: Proc<A>): ObservableK<A> =
+    ObservableK.async { _, cb -> fa(cb) }
+
+  override fun <A> asyncF(k: ProcF<ForObservableK, A>): ObservableK<A> =
+    ObservableK.asyncF { _, cb -> k(cb) }
+
+  override fun <A> asyncF(fa: ConnectedProcF<ForObservableK, A>): ObservableK<A> =
+    ObservableK.asyncF(fa)
+
+  override fun <A> CoroutineContext.startFiber(kind: ObservableKOf<A>): ObservableK<Fiber<ForObservableK, A>> =
+    asScheduler().let { scheduler ->
+      Observable.create<Fiber<ForObservableK, A>> { emitter ->
+        if (!emitter.isDisposed) {
+          val s: ReplaySubject<A> = ReplaySubject.create()
+          val conn: RxDisposable = kind.value().subscribeOn(scheduler).subscribe(s::onNext, s::onError)
+          emitter.onNext(Fiber(s.k(), ObservableK {
+            conn.dispose()
+          }))
+        }
+      }.k()
+    }
+
+  override fun <A> cancelableF(k: ((Either<Throwable, A>) -> Unit) -> ObservableKOf<CancelToken<ForObservableK>>): ObservableK<A> =
+    ObservableK.asyncF { kindConnection, function ->
+      k(function).map { kindConnection.push(it) }
+    }
+
+  override fun <A, B> CoroutineContext.racePair(fa: ObservableKOf<A>, fb: ObservableKOf<B>): ObservableK<RacePair<ForObservableK, A, B>> =
+    asScheduler().let { scheduler ->
+      Observable.create<RacePair<ForObservableK, A, B>> { emitter ->
+        val sa = ReplaySubject.create<A>()
+        val sb = ReplaySubject.create<B>()
+        val dda = fa.value().subscribe(sa::onNext, sa::onError)
```

Comment:
Any reason we don't complete here but we do in `raceTriple`?

Reply:
...a mistake. Having the complete is probably the correct version now that we use concatMap

Reply:
broke the build 😭